pyiode/iode_database/variables_database.pyx

```diff
@@ -899,6 +909,9 @@ cdef class Variables(_AbstractDatabase):
         to: str or Period, optional
             ending period to extrapolate variables.
             Defaults to the ending period if the current sample.
+        # GDM>
+        # * why not just 'variables'?
```

Comment:
because `variables` is the "global" (and unique) instance of the IODE Variables database.